### DISCUSSION TITLE
Build failing for Generic editor

### DISCUSSION THREAD

**Discussion by JeslyJoseph** [Sunbird-Ed/Community#562] [Installation] [answered]
Build using release tags: release-5.2.0

![Screenshot from 2023-07-03 11-11-02](https://github.com/Sunbird-Ed/Community/assets/111082725/52b50795-5be3-4c8a-a917-4f4ba29e5a65)
 I followed this link but still the build is failing
https://github.com/orgs/Sunbird-Ed/discussions/97#top

**Comment by vaibhavbhuva** [comment, marked as answer]
@JeslyJoseph  Thanks for reaching out.

From the above error message, it looks like you didn't install `libpng`, which is causing the error. Try installing `libpng` first and make sure you have all the prerequisites installed.

Here's the command:

`sudo apt-get install libpng-dev`

If you have made any previous npm installation attempts, which you most likely have, clean everything and then do a fresh dependency installation:

```
rm node_modules -R
rm package-lock.json
npm cache clear --force
npm install
```
Please let me know if you have any other questions.
cc: @vinukumar-vs @sajeshkayyath



Regards,
Vaibhav Bhuva

**Reply by JeslyJoseph** [reply]
Problem is solved by installing the above package
It's looks like a linux specific problem.
This would be useful for the future adopters, if updated in readme file as well
Thanks
